cli/waiter/subcommands/ping.py

```diff
@@ -1,87 +1,77 @@
 import logging
-import threading
-import time
-from enum import Enum
 
 from waiter import http_util, terminal
 from waiter.format import format_status
 from waiter.querying import query_token, print_no_data, query_service, get_services_using_token, get_service
-from waiter.util import guard_no_cluster, response_message, print_error, check_positive, wait_until, is_service_current
+from waiter.util import guard_no_cluster, print_error, check_positive, wait_until, is_service_current, response_message
```

Comment:
Nitpick: sorting in import statements.

I found [some backers](https://stackoverflow.com/a/20763446) of the idea :)

Reply:
Done